website/content/docs/enterprise/audit/filtering.mdx

```diff
@@ -168,3 +168,24 @@ The `/var/kv-audit.log` now includes four entries in total:
 
 The fallback device captured entries for the other commands. And the
  original audit file, `vault-audit.log`, continues to capture all audit events.
+
+
+<Tabs>
+
+  <Tab heading="Related concepts">
+    <ul>
+      <li>
+        <a href="/vault/docs/concepts/filtering">Filtering</a>
+      </li>
+    </ul>
```

Comment:
```suggestion
    
    - [Filtering](/vault/docs/concepts/filtering)

```

As long as you include whitespace before/after the `<Tab>` tag, you should be able to use markdown